.github/workflows/check_scala3_nightly.yml

```diff
@@ -17,7 +17,7 @@ jobs:
           echo "3.2.0-RC1-bin-20220307-6dc591a-NIGHTLY" > $BROKEN
           echo "3.2.0-RC1-bin-20220308-29073f1-NIGHTLY" >> $BROKEN
 
-          cs complete org.scala-lang:scala3-compiler_3: | grep NIGHTLY | grep -v -f $BROKEN | sort > all_nightly
+          cs complete-dep org.scala-lang:scala3-compiler_3: | grep NIGHTLY | grep -v -f $BROKEN | sort > all_nightly
```

Comment:
lol the most annoying change in the newer coursier. The amount of times I locally do `cs complete` still annoys me.

Reply:
haha
I took some time for me to realize that it's not a bug